str.h

```diff
@@ -9,7 +9,10 @@
 #define str_len(s) strlen((s))
 extern unsigned int str_chr();
 extern unsigned int str_rchr();
-extern int str_start();
+static inline int str_start(const char *s, const char *t)
+{
+  return strncmp(s, t, strlen(t)) == 0;
```

Comment:
I don't know which systems have `strncmp()` and which don't. How universally available is it?

Reply:
Possibly better as a `#define`?

Reply:
My manpage says:

```
CONFORMING TO
       POSIX.1-2001, POSIX.1-2008, C89, C99, SVr4, 4.3BSD.
```

Reply:
This is intentionally no define because with a inline function is is easy to be able to use "t" twice, with a macro one would have to add additional tricks to make it save that the expression isn't evaluated twice, for no real benefit.